src/background/background-process.js

```diff
@@ -343,16 +390,17 @@ export default class BackgroundProcess {
   updateBrowserActionForTab (tab) {
     if (tab && tab.hasOwnProperty('status')) {
       if (tab.isActive()) {
-        browser.browserAction.setTitle({title: BackgroundProcess.defaults.deactivateBrowserActionTitle, tabId: tab.tabID})
+        browser.browserAction.setTitle({title: BackgroundProcess.defaults.deactivateBrowserActionTitle, tabId: Tab.getTabIDfromUnique(tab.tabID)})
```

Comment:
If we would switch to using and storing `tab` instead of `tabID` everywhere, this would be simpler as we could just get tab ID as `tab.tabId`. I think that's really simpler than to reverse-engineer a unique tab ID. What do you think?

Reply:
Oh, thank you, Kirill for noticing - I have put tabObj inside tab to avoid using getTabIDfromUnique and didn't fix it here.